### PR TITLE
Fix golint failures of test/e2e/storage/testsuites

### PR DESCRIPTION
**Pull request by s-ito-ts**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
This fixes golint failures of the following file:
- test/e2e/storage/testsuites/base.go
- test/e2e/storage/testsuites/provisioning.go
- test/e2e/storage/testsuites/snapshottable.go
- test/e2e/storage/testsuites/testdriver.go

**Which issue(s) this PR fixes**:
Ref #68026

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

**Comment by SataQiu** [comment]
Thanks @s-ito-ts :+1: 
/LGTM

**Review by SataQiu — APPROVED** [review]

File: test/e2e/storage/testsuites/testdriver.go [hunk 1]
```diff
@@ -112,6 +112,7 @@ type SnapshottableTestDriver interface {
 // Capability represents a feature that a volume plugin supports
 type Capability string
 
+//Constants related to capability
```
Comment (draveness) [inline test/e2e/storage/testsuites/testdriver.go]: nit: need extra space after `//`

Comment (s-ito-ts) [reply test/e2e/storage/testsuites/testdriver.go]: Thanks for the review. I added the space.

**Review by draveness — APPROVED** [review]
/lgtm

**Comment by msau42** [comment]
/approve

Thanks!

**Comment by msau42** [comment]
/assign @spiffxp 
for approval

**Pull request merged**